styles/helpers/custom-media-queries.css

```diff
@@ -0,0 +1,69 @@
+/*---
+section: Helpers
```

Comment:
So I could create the documentation for how to use it.  Seems like a good place to put it.  Did you have somewhere better in mind to place it?